### ISSUE TITLE
0.38.1.0 Grashopper Grafted animaton

### ISSUE THREAD

**Issue by gesievi** [Archtica/Radii#105]
Hey thomas, i like the animation tool and it works. 

However its not very intuitive, i though i could just come at it like any grasshopper script and give it two lists of lists as in the picture below. 
Then Brep of (0;0) would be animated with planes of (0;0), Brep of (0;1) with the respective planes and so forth. I think "entwine"  feels like a detour. What was the rational behind this decision ? Maybe its just me that finds this counter intuitive.

![image](https://github.com/Archtica/RADii/assets/45074510/bc3d9376-7fe8-4547-974a-efb01fa6d184)

**Comment by Archtica** [comment]
Yes good point! I am aware of this and it is counter intuitive in a way. Unfortunately RADii output content is not graftable. It will mess with the correlation with the viewers that expects content per component.
I have thought about a solution, but it will take a lot to reconfigure, but It's on the list!

**Comment by gesievi** [comment]
In this case i would not prioritise it, as it is working currently.

**Comment by Archtica** [comment]
I gave it some more thought and I think I have a realative easy solution. I think all radii components shoiuld be able to output grafted content. That means we can save components rather than one per single collection. I will work on it for a later release. I'm just abouyt ready to ship the next beta that has the new PublishReference component.
